lib/jsdom/browser/default-stylesheet.js

```diff
@@ -128,14 +123,6 @@ q {
     display: inline
 }
 
-q:before {
-    content: open-quote;
```

Comment:
@domenic 
now nwmatcher does support the :after and :before pseudo-elements and a few others too, the supports is obviously limited to their parsing (grammar) without throwing exceptions.

For these pseudo-elements, the `select()` method will return an empty node set (empty array) while the `match()` method will always return `false`.